content/generic.rst

```diff
@@ -21,6 +21,9 @@ something is wrong.
 
 * Rule #3: If it ain't broke, don't fix it!
 
+* Ammendment to Rule #3: If it's not totally broken but still wastes a lot of your time, fix it! 
+  This means all the FIXME tags should be a priority when 'refactoring <http://ronjeffries.com/xprog/articles/refactoring-not-on-the-backlog/>'.
```

Comment:
Yes, that's also my understanding and I'll rephrase to get the correct idea through.